src/styles/StyleUtils.js

```diff
@@ -210,13 +226,10 @@ function getBackgroundColorStyle(backgroundColor) {
  * @returns {Object}
  */
 function getBackgroundColorWithOpacityStyle(backgroundColor, opacity) {
-    const result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(backgroundColor);
+    const result = hexadecimalToRGB(backgroundColor);
     if (result !== null) {
-        const r = parseInt(result[1], 16);
-        const g = parseInt(result[2], 16);
-        const b = parseInt(result[3], 16);
         return {
-            backgroundColor: `rgba(${r}, ${g}, ${b}, ${opacity})`,
+            backgroundColor: `rgba(${result[1]}, ${result[1]}, ${result[1]}, ${opacity})`,
```

Comment:
I don't think these indexes are right, are they? Shouldn't they be 0, 1, 2? I kind of think that `hexidecimalToRGB` isn't quite right. What I would expect to see is something more like `hexidecimalToRGBA` and it just returns a single string that is in the format `"rgba(blah blah)"`

Reply:
Agreed, the indices should be `0`, `1`, and `2`. I do not think that `hexidecimalToRGB` should return a string (i.e., `rgba(...)`) because it would require us to convert the string to the RGB components anyway. However, I agree that the method should communicate its intent more clearly. Will rename `hexidecimalToRGB` to `hexadecimalToRGBComponents`.